flathub.json

```
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["aarch64"]
```

```suggestion
  "only-arches": ["x86_64"]
```

still.... why not supporting it? I thought that .Net was supposed to be cross platform.

---

Tomorrow I am going to Japan for 3 weeks.

When I come back, I will try to make it work for `aarch64`.

Right now https://github.com/Jinjinov/flathub/blob/net.openhabittracker.app/nuget-sources.json has only `linux-x64` NuGet packages.